gradle/repositories.gradle

```diff
@@ -0,0 +1,23 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+repositories {
+    jcenter()
+    mavenCentral()
+    gradlePluginPortal()
```

Comment:
@vidakovic what is this `gradlePluginPortal()` required for? We don't currently seem to use it in our `fineract-provider/build.gradle`. If you need it later, it's probably clearer add it only when it's required, not yet here? But even so.. my Gradle foo is quite limited, but if anything shouldn't this only be in `buildscript { repositories` and never root `repositories`, because Gradle Plugins should never be "runtime" dependencies? Is `gradlePluginPortal()` the equivalent of ``maven { url "https://plugins.gradle.org/m2/" }`?  @ptuomola